.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: Test
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
This ensures that this workflow is run automatically for each pull request and when the PR is merged to main